### PR TITLE
Closes AgileVentures/MetPlus_tracker#292

### PR DESCRIPTION
**Pull request by sandhyakoti**

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -20,6 +20,8 @@
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/jquery-noty/2.3.8/themes/bootstrap.min.js'
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.2/js/select2.full.min.js'
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/spin.js/2.3.2/spin.min.js'
+    = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.3/moment.min.js'
+    = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.7/moment-timezone-with-data.min.js'
```
Comment (joaopapereira) [inline app/views/layouts/application.html.haml]: Ignore this comment

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -20,6 +20,8 @@
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/jquery-noty/2.3.8/themes/bootstrap.min.js'
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.2/js/select2.full.min.js'
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/spin.js/2.3.2/spin.min.js'
+    = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.3/moment.min.js'
```
Comment (joaopapereira) [inline app/views/layouts/application.html.haml]: Ignore this comment

File: app/views/job_seekers/_form.html.haml [hunk 3]
```diff
@@ -29,10 +29,8 @@
     .clearfix
     .col-sm-8.col-sm-offset-2
       - if @current_resume
-        = "(Your resume on file is \"#{@current_resume.file_name}\",       |
-                 and was uploaded on #{@current_resume.updated_at.         |
-                in_time_zone('Eastern Time (US & Canada)').to_s(:long)})"  |
-
+        = "Your resume on file is \"#{@current_resume.file_name}\", and was uploaded on"
+        %span.utc_to_local_time #{@current_resume.updated_at}
```
Comment (patmbolger) [inline app/views/job_seekers/_form.html.haml]: This comment is a matter of personal preference, so safe to ignore IMO.

Comment (patmbolger) [inline app/views/job_seekers/_form.html.haml]: Actually, I just saw João's comment requesting that this change be made .... please ignore my comment (as well as similar below).

**Comment by joaopapereira** [comment]
@sandhyakoti Any idea when you will be able to handle the suggestions of Hound?

**Pull request merged**